src/devTools.js

```diff
@@ -136,6 +139,7 @@ function liftReducer(reducer, initialState) {
       if (currentStateIndex === stagedActions.length - 1) {
         currentStateIndex++;
       }
+      liftedAction.action.devTimestamp = Date.now();
```

Comment:
Shouldn't this be part of `liftedAction` itself? This is what it's for: to carry the information that makes sense to the lifted reducer. 

How about changing

```js
function liftAction(action) {
  const liftedAction = { type: ActionTypes.PERFORM_ACTION, action };
  return liftedAction;
```

to

```js
function liftAction(action) {
  const liftedAction = {
    type: ActionTypes.PERFORM_ACTION,
    timestamp: Date.now(),
    action
  };
  return liftedAction;
```

instead?

If you do it this way, you can change the lifted reducer to extract timestamps into a separate `actionTimestamps` lifted state field, which dev tools can access.